src/NzbDrone.Core/Download/DownloadService.cs

```diff
@@ -48,10 +48,18 @@ public class DownloadService : IDownloadService
             _logger = logger;
         }
 
-        public async Task SendReportToClient(ReleaseInfo release, string source, string host, bool redirect)
+        public void SendReportToClient(ReleaseInfo release, string source, string host, bool redirect, int? downloadClientId)
+        {
+            var downloadClient = downloadClientId.HasValue
```

Comment:
This will always be true because `downloadClientId` is 0 by default (any). See the SendReportToClient function calls in SearchController where `indexerDef.DownloadClientId` is always passed as an argument.

That's the reason why I did `downloadClientId > 0` instead.